addons/account/tests/test_account_move_out_invoice.py

```diff
@@ -1977,6 +1977,63 @@ def test_out_invoice_create_refund(self):
             'payment_state': 'paid',
         })
 
+    def test_out_invoice_create_refund_qr_code(self):
+        self.invoice.qr_code_method = 'sct_qr'
+        self.invoice.action_post()
+
+        move_reversal = self.env['account.move.reversal'].with_context(active_model="account.move", active_ids=self.invoice.ids).create({
+            'date': fields.Date.from_string('2019-02-01'),
+            'reason': 'no reason',
+            'refund_method': 'refund',
+        })
+        reversal = move_reversal.reverse_moves()
+        reverse_move = self.env['account.move'].browse(reversal['res_id'])
+
+        self.assertEqual(self.invoice.payment_state, 'not_paid', "Refunding with a draft credit note should keep the invoice 'not_paid'.")
+        self.assertInvoiceValues(reverse_move, [
+            {
+                **self.product_line_vals_1,
+                'amount_currency': 1000.0,
+                'debit': 1000.0,
+                'credit': 0.0,
+            },
+            {
+                **self.product_line_vals_2,
+                'amount_currency': 200.0,
+                'debit': 200.0,
+                'credit': 0.0,
+            },
+            {
+                **self.tax_line_vals_1,
+                'amount_currency': 180.0,
+                'debit': 180.0,
+                'credit': 0.0,
+            },
+            {
+                **self.tax_line_vals_2,
+                'amount_currency': 30.0,
+                'debit': 30.0,
+                'credit': 0.0,
+            },
+            {
+                **self.term_line_vals_1,
+                'name': '',
+                'amount_currency': -1410.0,
+                'debit': 0.0,
+                'credit': 1410.0,
+                'date_maturity': move_reversal.date,
+            },
+        ], {
+            **self.move_vals,
```

Comment:
Either you do this to stay consistent or you do `}, {` on the previous line
```suggestion
            },
        ], 
        {
            **self.move_vals,
```